cicecore/cicedyn/general/ice_init.F90

```diff
@@ -1570,6 +1574,10 @@ subroutine input_data
          abort_list = trim(abort_list)//":24"
       endif
 
+      if(trim(hist_time_axis) /= 'begin' .and. trim(hist_time_axis) /= 'middle' .and. trim(hist_time_axis) /= 'end') then
+         write (nu_diag,*) subname//' WARNING: hist_time_axis set to default end '
```

Comment:
I would abort here.  Change to something like

```
         write (nu_diag,*) subname//' ERROR: hist_time_axis value not valid = '//trim(hist_time_axis)
         abort_list = trim(abort_list)//":24"
```

Instead of 24, give it a unique number, can quickly great for abort_list to find one.